packages/scaffold-html/src/views/w3m-what-is-a-network-view/index.ts

```diff
@@ -0,0 +1,73 @@
+import { LitElement, html } from 'lit'
+import { customElement } from 'lit/decorators.js'
+import styles from './styles'
+import { VisualType } from '@web3modal/ui/src/utils/TypesUtil'
+import { RouterController } from '@web3modal/core'
+
+type Data = {
+  images: VisualType[]
+  title: string
+  text: string
+}
+
+const data: Data[] = [
+  {
+    images: ['network', 'layers', 'system'],
+    title: 'The system’s nuts and bolts',
+    text: 'A network is what brings the blockchain to life, as this technical infrastructure allows apps to access the ledger and smart contract services.'
+  },
+  {
+    images: ['noun', 'defiAlt', 'dao'],
+    title: 'Designed for different uses',
+    text: 'A wallet lets you store, send and receive digital assets like cryptocurrencies and NFTs.'
+  }
+]
+
+@customElement('w3m-what-is-a-network-view')
+export class W3mWhatIsANetworkView extends LitElement {
+  public static styles = styles
+
+  // -- Render -------------------------------------------- //
+
+  public render() {
+    return html`
+      <wui-flex
```

Comment:
Looks like we are repeating same bit of code here as with another help view, perhaps we can make a partial i.e. `w3m-help-widget` that takes in params like data array and onLearnMore function? We'll likely need another optional param like `onAction` for that "Get wallet" button latter on.